src/molecules/Tabs/Tabs.native.js

```diff
@@ -76,10 +76,12 @@ const Tabs = ({ children, defaultValue, value, onChange }) => {
 
   const onIndexChange = useCallback(
     (newIndex) => {
-      if (!isDefined(value)) {
-        setIndex(newIndex);
+      if (typeof newIndex === 'number') {
```

Comment:
why typeOf check?

Reply:
During jest tests it returns an object with a function in the callback instead of an index. Couldn't identify what the problem is with the library but this adds a safeguard so that even during tests the components work as intended